includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1777,8 +1777,8 @@ private function get_intent( $intent_type, $intent_id ) {
 
 		if ( $response && isset( $response->{ 'error' } ) ) {
 			$error_response_message = print_r( $response, true );
-			WC_Stripe_Logger::log( "Failed to get Stripe intent $intent_type/$intent_id." );
-			WC_Stripe_Logger::log( "Response: $error_response_message" );
+			WC_Stripe_Logger::error( "Failed to get Stripe intent $intent_type/$intent_id." );
+			WC_Stripe_Logger::error( "Response: $error_response_message" );
```

Comment:
I think we should log only one entry here, largely so we can avoid having duplicate stack traces.
```suggestion
			WC_Stripe_Logger::error( "Failed to get Stripe intent $intent_type/$intent_id.", [ 'response' => $response ] );
```